detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/cli/MavenCliExtractor.java

```diff
@@ -46,13 +49,27 @@ public Extraction extract(File directory, ExecutableTarget mavenExe, MavenCliExt
         List<String> includedScopes = mavenCliExtractorOptions.getMavenIncludedScopes();
         List<String> excludedModules = mavenCliExtractorOptions.getMavenExcludedModules();
         List<String> includedModules = mavenCliExtractorOptions.getMavenIncludedModules();
+        Boolean includeShadedDependencies = mavenCliExtractorOptions.getMavenIncludeShadedDependencies();
+
+        if(includeShadedDependencies) {
+            try {
+                mavenProjectInspectorDetectable.includeShadedDependencies = "include_shaded_dependencies";
```

Comment:
The "include_shaded_dependencies" string is used in several places. It would be good to have a constant for it.

Reply:
Also, I wonder if we should pass around a boolean instead of the string in all cases except in one where it really needs to be a string for PI.